src/main/java/tictactoe/PlayerFactory.java

```diff
@@ -9,7 +11,7 @@
     }
 
     public Player createNewPlayer(String playerType, String mark) {
-        if (playerType.equals("h")) {
+        if (playerType.equals(PlayerTypes.HUMAN.type)) {
```

Comment:
Is the `type` field needed?

Reply:
If the changes above are implemented (expecting the user to enter `human` or `computer` when selecting the type of player), `type` is not needed and can be replaced with `to.String()`